packages/core/src/State.ts

```diff
@@ -90,7 +90,7 @@ interface MachineSnapshotBase<
   /**
    * The enabled state nodes representative of the state value.
    */
-  configuration: Array<StateNode<TContext, TEvent>>;
+  nodes: Array<StateNode<TContext, TEvent>>;
```

Comment:
while we are at it I'd propose to rename this to `_nodes`

Reply:
Sure - go for it